service/application/pom.xml

```diff
@@ -15,7 +15,6 @@
     <description>event-management-agent application</description>
 
     <properties>
-        <camel.version>3.19.0</camel.version>
```

Comment:
Can you please explain the relationships between these pom files and which of them will provide the value for ${camel.version} ?

Can all these pom files get their camel.version from one pom file only? (this question can be invalid depending on the answer to the former question)

Reply:
There is a top-level pom.xml file, and then each directory under the top level has it's own pom.xml. If the subdirectory pom.xml has a parent section, then it inherits the properties of the top-level pom.xml.

In this case the local-storage-plugin/pom does not have a parent section pointing to the top-level pom.xml, so I need to change the version in its pom.xml. The other two subdirectories (application and plugin) inherit from the top-level pom.

@slunelsolace Can we add a parent section to the local-storage-plugin/pom.xml to point to the top-level pom or was that omitted by design?

Reply:
@gregmeldrum It's by design. If a project doesn't need the parent dependency it shouldn't be required to import it. Later on these modules should be in their own repositories. I don't think we should be tightly coupling them or decoupling them could be a pain later.

I'd be more in favor are starting an initiative to get plugins out of this repository.